sql/payday.sql

```diff
@@ -76,12 +76,27 @@ CREATE INDEX ON payday_payment_instructions (participant);
 CREATE INDEX ON payday_payment_instructions (team);
 ALTER TABLE payday_payment_instructions ADD COLUMN is_funded boolean;
 
+UPDATE payday_payment_instructions ppi
+   SET due = s.due
+  FROM (SELECT participant, team, SUM(due) AS due
```

Comment:
Gotcha. I think we should update the `due` column every time a new `payment_instruction` is `INSERT`ed. We want to be able to surface that data in the UI, so we should keep it current, we shouldn't only ever update it during payday.

> Thought of using an AFTER INSERT trigger[.]

How about carrying the `due` column forward during `Participant.set_payment_instruction`?